.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -35,5 +35,9 @@ Paste the output from `argocd version` here.
 Paste any relevant application logs here.
 ```
 
-**Additional context**
-Add any other context about the problem here.
+**Have you thought about contributing a fix yourself?**
+
+Open Source software thrives with your contribution. It not only gives skills you might not be able to get in your day job, it also looks amazing on your resume. 
+
+If you want to get involved, check out the 
+[contributing guide](https://github.com/argoproj/argo-cd/blob/master/docs/CONTRIBUTING.md), then reach out to us on [Slack](https://argoproj.slack.com/messages/CASHNF6MS) so we can see how to get you started.
```

Comment:
It might be better to use slack URL with invite token https://argoproj.github.io/community/join-slack